homeassistant-supervised/DEBIAN/preinst

```diff
@@ -10,6 +10,18 @@ warn "Home Assistant as a VM or run Home Assistant Core"
 warn "via a Docker container."
 warn ""
 
+# Check if we are running on a supported OS
+CURRENT_OS=$(lsb_release -d)
+if [[ $CURRENT_OS != *"Debian GNU/Linux 11 (bullseye)"* ]]; then
```

Comment:
Should we add an env var to override it?
e.g., as soon as new Debian land, you want to be able to force install it, e.g., for testing.

Reply:
Yes, I agree